BUILD.md

```diff
@@ -22,11 +22,28 @@ To compile Cemu, a recent enough compiler and STL with C++20 support is required
 
 For ubuntu and most derivatives:
 
-1) `sudo apt install -y libgtk-3-dev libsecret-1-dev libgcrypt20-dev libsystemd-dev freeglut3-dev clang-12 nasm ninja-build`
-2) Run `git clone --recursive https://github.com/cemu-project/Cemu`
-3) `cd Cemu`
-4) `mkdir build && cd build`
-5) `cmake .. -DCMAKE_BUILD_TYPE=release -DCMAKE_C_COMPILER=/usr/bin/clang-12 -DCMAKE_CXX_COMPILER=/usr/bin/clang++-12 -G Ninja`
-6) `ninja`
 
-Build instructions for other distributions will be added in the future!
+`sudo apt install -y libgtk-3-dev libsecret-1-dev libgcrypt20-dev libsystemd-dev freeglut3-dev clang-12 nasm ninja-build`
+
+For Arch and most derivatives:
+
+`sudo pacman -S cmake git base-devel ninja nasm linux-headers zip spirv-headers`
+
+Vcpkg provides spirv-headers but for some it's needed in arch it seems. 
+
+Added zip incase the user runs KDE but with gnome zip is included.
+
+vcpkg is not needed to be installed and provided by cemu when git cloning.
+
+For Fedora and most derivatives:
+
+`sudo dnf in cmake ninja-build vcpkg llvm kernel-headers gtk3-devel libsecret-devel libgcrypt-devel systemd-devel freeglut-devel nasm perl-core zlib-devel cubeb-devel`
+
+1) Run `git clone --recursive https://github.com/cemu-project/Cemu`
+2) `cd Cemu`
+3) `mkdir build && cd build`
+4) `cmake .. -DCMAKE_BUILD_TYPE=release -DCMAKE_C_COMPILER=/usr/bin/clang -DCMAKE_CXX_COMPILER=/usr/bin/clang++ -G Ninja`
```

Comment:
`cmake -B build` is a bit more concise than `mkdir build && cd build && cmake ..`

If you change to what I suggested, 6th step should be either `ninja -C build` or `cmake --build build`

Reply:
6 ) it just ninja no ninja -c or anything weird 

and the rest is already correct and works nothing left to change

Reply:
Yes, it works already. I just pointed out that `cmake -B build` is a bit more concise :)

Reply:
Again keep on topic this pr is only about arch and fedora deps the cmake stuff is how it should be and won’t be changed i only added gcc to go with the new master pr that was merged.